### ISSUE TITLE
pinned

### ISSUE THREAD

**Issue by jrr45** [mbnuqw/sidebery#111]
## Environment info

OS:  
Firefox version:  
Addon version:  

<details><code>

</code></details>

**Comment by Tylast** [comment]
I think I have the same problem.  After I open a new tab from a pinned tab.  I go to that new tab...then close it.  I am not taken back to that pinned tab (which was the last tab I was at).  If the tab is not pinned, it works fine.

**Comment by mbnuqw** [comment]
@Tylast, yes, unfortunately current mechanism of finding the next active tab have a few faulty edge cases. I plan to rewrite it soon<sub>er or later</sub>.
